### PR TITLE
Add rendering of type property descriptions

### PR DESCRIPTION
**Pull request by DominikaK**
Adds rendering of the descriptions of type properties:
image

To do: 
- [x] Add generated output file

### CODE REVIEW

File: raml2html/themes/default/type.html.twig [hunk 1]
```diff
@@ -4,12 +4,17 @@
     <span class="red-text">*</span>
 {% endif %}
 
-:
-<span class="blue-text">
-    {% if type is array type %}
-        {{ type.items.name }}[]
+(<span class="type-type">
```
Comment (ciastektk) [inline raml2html/themes/default/type.html.twig]:
```suggestion
(<span class="font-italic">
```

File: assets/css/style.css [hunk 2]
```diff
@@ -386,3 +386,11 @@ h5[data-field="name"]:before,
 .navbar-toggler-icon {
     background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(0, 0, 0, 0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E");
 }
+
```
Comment (ciastektk) [inline assets/css/style.css]: Below code should be removed. You can use bootstrap classes to set text transformation e.g.:  `font-italic`, `font-weight-bold` etc.

**Review by ciastektk — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 2 above)
Comment (Steveb-p) [reply assets/css/style.css]: You can select relevant lines by shift-clicking the line numbers @ciastektk 

image

Comment (ciastektk) [reply assets/css/style.css]: Thanks @Steveb-p! It will be very useful 😄

**Review by ciastektk — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**